intro-to-R-tidyverse/03-intro_to_r_exercise.Rmd

```diff
@@ -0,0 +1,197 @@
+---
+title: 'Exercise: Getting comfortable with R'
+output:
+  html_notebook:
+    toc: yes
+    toc_float: yes
+---
+
+The goal of these exercises is to help you get comfortable with using R and R 
+notebooks. 
+
+We will create a play data set for an imaginary mouse experiment where you have 
+control and treatment groups of mice. 
+
+### Create a imaginary mouse experiment
+
+Use the empty chunk below to create a *character* vector with made up sample IDs 
+for your mice in your imaginary experiment.
+Call this new vector `mouse_id` and make sure each ID is distinct.  
+
+```{r}
+mouse_id <-
+```
+
+Check your environment panel. Is `mouse_id` there? 
+
+Use the empty code chunk below to evaluate the structure and type of your object.
+
+```{r}
+
+```
+
+Now evaluate the `length` of `mouse_id`. 
+Remember you can look up information about a function using `?`.
+
+```{r}
+
+```
+
+Now use the `unique` function to check if your `mouse_id` vector has all unique
+ids. 
+Print out the original `mouse_id` vector and the unique `mouse_ids`s and compare 
+them. 
+
+```{r}
+
+```
+
+Now make a *factor* vector that is the same length as `mouse_id` but contains 
+the experiment group membership information for each mouse (e.g. "control").  
+- Call this vector `exp_group`.  
+- Stick to 2 or 3 experimental groups.  
+
+To make this, start with a character vector that has values repeated and then 
+use the `as.factor` function. 
+
+```{r}
+
+```
+
+Use the function `summary` to look at your factor vector. 
+How many experimental groups do you have?  
+
+```{r}
+
+```
+
+Now use the `length` function and the `==` operator to check if your `mouse_id` 
+and `exp_group` vectors are the same length. 
+
+```{r}
+
+```
+
+Use `levels` to explore the order of the factor levels of `exp_group`. 
+
+```{r}
+
+```
+
+Do the levels make sense? Is your control the first level? 
+If not, create a a new code chunk here and re-order your levels. 
+Look up the `levels` function using `?` to do this. 
```

Comment:
This may lead participants to try `levels?` instead of `?levels`. While allowing them to figure out some things on their own is a part of the process,  it may help to be more explicit here. What are your thoughts?